<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockWidget</class>
 <widget class="QWidget" name="StockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Stock</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelDate">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelMetal">
     <property name="text">
      <string>Metal</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboMetal">
     <item>
      <property name="text">
       <string>Gold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Silver</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Platinum</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDetail">
     <property name="text">
      <string>Detail</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leDetail"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelNote">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leNote"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelVoucher">
     <property name="text">
      <string>Voucher No</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leVoucher"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelPurity">
     <property name="text">
      <string>Purity</string>
     </property>
     <property name="styleSheet">
       <string>background-color: yellow;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spinPurity">
     <property name="maximum">
      <double>100.0</double>
     </property>
     <property name="styleSheet">
       <string>background-color: yellow;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelWeight">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spinWeight">
     <property name="maximum">
      <double>999999.99</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label24k">
     <property name="text">
      <string>24K</string>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="le24k">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelPrice">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="spinPrice">
     <property name="maximum">
      <double>99999999.99</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelAmount">
     <property name="text">
      <string>Amount</string>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="leAmount">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayoutBtns">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
